src/db/migrations/20190825153510-create-requests.js

```diff
@@ -0,0 +1,49 @@
+module.exports = {
+  up: (queryInterface, Sequelize) => (
+    queryInterface.createTable('requests', {
+      id: {
+        allowNull: false,
+        autoIncrement: true,
+        primaryKey: true,
+        type: Sequelize.INTEGER
+      },
+      user_id: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        references: {
+          model: 'users',
+          key: 'id',
+          as: 'user_id'
+        },
+        onDelete: 'CASCADE'
+      },
+      trip_id: {
+        type: Sequelize.INTEGER,
+        allowNull: false,
+        references: {
+          model: 'trips',
+          key: 'id',
+          as: 'trip_id'
+        },
+        onDelete: 'CASCADE'
+      },
+      reason: {
```

Comment:
Could you please explain why the requests table is needed? Because as it stands, you only moved a few columns from the trips table here. After that you referenced the trips table. If we are to join both the requests and trips table we will have exactly the same columns that were in the original trips table. Therefore, the reason for the split becomes unclear.

Reply:
1. The PT said `Implement a requests table so that.....` . Emphasis on `implement` plus `table`
2. The splitting of your DB into small partitions; otherwise known as `sharding` is an industry-accepted way of reducing load and making for faster sql transactions plus better management.

Reply:
Like I previously commented, would it have made any difference if `trips` was renamed to `requests`?

Reply:
Trips are the actual `planned movement from point A to B`.  Requests are the `reasons and plea` for such trips to happen. Two distinct models. Plus ideally, `A request should be granted, before a trip can be made`. A rejected request nullifies a trip. You can choose to nullify the whole `trip row` if it were integrated as one. Or you can choose to nullify just the `request row` if it were abstracted. Request table = lesser rows, Trips table -= more rows. Speed vs Compactness. Speed wins all the time.

Reply:
Plus the PT says `implement a Request Table` and NOT `integrate a request logic into the trips table`